content/SCALE/SCALETutorials/DataProtection/TrueCloudTasks.md

```diff
@@ -0,0 +1,273 @@
+---
+title: "Managing TrueCloud Backup Tasks"
+description: "Provides instructions on how to set up a TrueCloud Backup Task and how to configure an Storj iX account to work with TrueNAS."
+weight: 5
+tags:
+- TrueCloud
+- cloud
+- backup
+keywords:
+- nas cloud storage
+- data protection
+- data backup and recovery
+aliases: 
+- cloudsynctasks/addstorjcloudsynctask
+---
+
+TrueNAS can send, receive, or synchronize data with the cloud storage provider Storj.
+TrueCloud Backup Tasks allow for single-time transfers or recurring transfers on a schedule.
+They are an effective method to back up data to a remote location.
+
+{{< hint type=important >}}
+To take advantage of the lower-cost benefits of the TrueCloud backup service, you must create your Storj iX account using the link provided on the **Add Cloud Credentials** screen.
+
+You must also create and authorize the storage buckets on Storj for use by SCALE.
+
+iXsystems is not responsible for any charges you incur using a third-party vendor with the TrueCloud Backup feature.
+{{< /hint >}}
+
+This procedure provides instructions to set up both Storj and SCALE.
+
+## TrueCloud Backup Task Requirements
+
+You must have all system storage (pool and datasets or zvols) configured and ready to receive or send data.
+
+## Creating a TrueCloud Backup Task
+
+To create a TrueCloud Backup task for a TrueCloud transfer:
+
+1. Create the SCALE [cloud credential](#adding-storj-cloud-credentials).
+
+   Adding the cloud credential in SCALE includes using the link to create the Storj iX account, create a new bucket, and obtain the S3 authentication credentials needed to complete the process in SCALE.
+
+2. Create the [Storj iX account](#creating-the-storj-truenas-scale-account).
+
+   You must create a new Storj iX account to use SCALE to access a Storj account.
+
+3. Add a  new [Storj bucket](#adding-the-storj-truenas-bucket).
+
+4. [Create Storj S3 access](#setting-up-s3-access-to-the-bucket) for the new bucket.
+
+5. Finish creating the SCALE [cloud credential](#adding-storj-cloud-credentials) using the S3 access and secret keys provided by Storj.
+
+6. Create the [TrueCloud Backup task](#setting-up-the-truecloud-backup-task) for one bucket.
```

Comment:
The details in the "substeps" below step 1 can be eliminated here as the details of actually creating the cloud credential cover it. 
```suggestion
1. Create the SCALE [cloud credential](#adding-storj-cloud-credentials).

   Adding the cloud credential in SCALE includes using the link to create the Storj iX account, create a new bucket, and obtain the S3 authentication credentials needed to complete the process in SCALE.

2. Create the [TrueCloud Backup task](#setting-up-the-truecloud-backup-task) for one bucket.
```